Resources/Prototypes/Recipes/Construction/Graphs/structures/girder.yml

```diff
@@ -358,54 +383,90 @@
+        - to: reinforcedWallStage1
+          steps:
             - tool: Screwing
-              doAfter: 1
+              doAfter: 4
+
+    - node: reinforcedWallStage1
+      entity: WallReinforcedS1
+      edges:
+        - to: reinforcedWallStage0
+          steps:
+            - tool: Screwing
+              doAfter: 4
+        - to: reinforcedWallStage2
+          steps:
             - tool: Welding
-              doAfter: 5
-              completed:
-                - !type:VisualizerDataInt
-                  key: "enum.ReinforcedWallVisuals.DeconstructionStage"
-                  data: 4
+              doAfter: 4
+
+    - node: reinforcedWallStage2
+      entity: WallReinforcedS2
+      edges:
+        - to: reinforcedWallStage1
+          steps:
+            - tool: Welding
+              doAfter: 6
+        - to: reinforcedWallStage3
+          steps:
             - tool: Prying
-              doAfter: 1
-            - tool: Anchoring
-              doAfter: 1
-              completed:
-                - !type:VisualizerDataInt
-                  key: "enum.ReinforcedWallVisuals.DeconstructionStage"
-                  data: 3
+              doAfter: 10
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetPlasteel1
+              amount: 1
+
+    - node: reinforcedWallStage3
+      entity: WallReinforcedS3
+      edges:
+        - to: reinforcedWall
+          steps:
+          - material: Plasteel
+            amount: 1
+            doAfter: 5
+        - to: reinforcedWallStage4
+          steps:
+          - tool: Anchoring
+            doAfter: 4
+
+    - node: reinforcedWallStage4
+      entity: WallReinforcedS4
+      edges:
+        - to: reinforcedWallStage3
+          steps:
+          - tool: Anchoring
+            doAfter: 4
+        - to: reinforcedWallStage5
+          steps:
+          - tool: Welding
+            doAfter: 10
+
+    - node: reinforcedWallStage5
+      entity: WallReinforcedS5
+      edges:
+        - to: reinforcedWallStage4
+          steps:
             - tool: Welding
               doAfter: 10
-              completed:
-                - !type:VisualizerDataInt
-                  key: "enum.ReinforcedWallVisuals.DeconstructionStage"
-                  data: 2
+        - to: reinforcedGirder
+          steps:
             - tool: Prying
-              doAfter: 1
-              completed:
-                - !type:VisualizerDataInt
-                  key: "enum.ReinforcedWallVisuals.DeconstructionStage"
-                  data: 1
-            - tool: Screwing
-              doAfter: 1
-              completed:
-                - !type:VisualizerDataInt
-                  key: "enum.ReinforcedWallVisuals.DeconstructionStage"
-                  data: 0
-            - tool: Cutting
-              doAfter: 1
+              doAfter: 10
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetPlasteel1
+              amount: 1
```

Comment:
ive never actually realised how long it takes to deconstruct walls in ss13 but jeeesuuuus maybe nerf their health so structural damage can actually do its thing

Reply:
i'll go over damage thresholds and compare with our existing behaviour